docs/src/app/components/pages/components/left-nav.jsx

```diff
@@ -154,33 +196,49 @@ export default class LeftNavPage extends React.Component {
         <CodeExample code={Code}>
           <div>
             <div>
-              <RaisedButton label="Toggle Docked Left Nav" onTouchTap={this._toggleDockedLeftNavClick} /><br/><br/>
-              <RaisedButton label="Show Hideable Left Nav" onTouchTap={this._showLeftNavClick} /><br/><br/>
-              <RaisedButton label="Show Hideable Children Left Nav" onTouchTap={this._showLeftNavChildrenClick} /><br/><br/>
+              <RaisedButton label="Swipe to View Uncontrolled Left Nav" disabled={true} /><br/><br/>
```

Comment:
Well, it's there for the people visiting this page from desktop PC. I for one had no idea there was a LeftNav until I opened the dev tools and accidentally swiped right and saw it popping up. If I'm gonna remove this example we have to find a way to inform those users that there actually is an uncontrolled LeftNav right there in the doc site. And that one is the only way an uncontrolled LeftNav can be demonstrated.